rpc/src/rpc.rs

```diff
@@ -413,7 +413,7 @@ impl JsonRpcRequestProcessor {
                 })
                 .collect::<Result<Vec<_>>>()?
         };
-        Ok(result).map(|result| match with_context {
+        Ok(match with_context {
             true => OptionalContext::Context(new_response(&bank, result)),
             false => OptionalContext::NoContext(result),
```

Comment:
Do you mind renaming this variable `result`, while you're in here?
That word is overloaded, so it's a little confusing. Maybe something like `accounts` instead?